src/Domain/Composer/Version/DrupalCoreVersionResolver.php

```diff
@@ -220,6 +220,16 @@ private function findLatestLts(): string {
 
     $parts = explode('.', $this->findCurrent());
     $current_major = array_shift($parts);
+    // Drupal 8 is still marked as supported in the API even though it's EOL.
+    // @todo remove this block once D8 is EOL in the updates API.
+    if ($current_major === 9) {
+      $message = sprintf(
+        'No Drupal core version satisfies the given constraints: version=%s, stability=%s',
+        9,
```

Comment:
I know the sprintf is unnecessary, but I copied this code block from `$this->resolveArbitrary()` just for consistency.